extensions/mentions/js/src/forum/utils/getMentionText.js

```diff
@@ -28,9 +32,12 @@ export default function getMentionText(user, postId) {
     // @"Display name"#UserID
     const cleanText = getCleanDisplayName(user);
     return `@"${cleanText}"#${user.id()}`;
-  } else {
+  } else if (group === undefined) {
     // @"Display name"#pPostID
     const cleanText = getCleanDisplayName(user);
     return `@"${cleanText}"#p${postId}`;
+  } else if (postId === undefined) {
+    // @"Name Plural"#gGroupID
+    return `@"${group.namePlural()}"#g${group.id()}`;
```

Comment:
I'd prefer changing the conditionals to `if user, else if post, else if group, else exception` it's more readable and it takes into consideration the case where everything is undefined/falsy.